go/libraries/doltcore/sqle/logictest/DoltRegressionsJenkinsfile

```diff
@@ -0,0 +1,91 @@
+pipeline {
+    agent none
+    stages {
+        stage('TestSQL') {
+            parallel {
+                stage ("update Liquidata/dolt-sql-performance") {
+                    agent {
+                        kubernetes {
+                            label "liquidata-inc-ld-build"
+                        }
+                    }
+                    environment {
+                        PATH = "${pwd()}/.ci_bin:${pwd()}/.ci_bin/node_modules/.bin:${env.PATH}"
+                        DOLT_VERSION = sh(returnStdout: true, script: "(git rev-parse HEAD | tr -d '\n')")
+                        TMPDIR = "${pwd()}/tempDir"
+                        CREDSDIR = "/home/jenkins/.dolt/creds"
+                        DOLT_CREDS = credentials("system-account-dolthub-creds")
+                        TEST_N_TIMES = 5 // like to pass this from the Jenkins UI?
+                        DOLT_GLOBAL_CONFIG = "/home/jenkins/.dolt/config_global.json"
+                    }
+                    steps {
+                        dir ("sqllogictest") {
+                            git url: "https://github.com/liquidata-inc/sqllogictest.git"
+                        }
+                        dir ("go") {
+                            sh "rm -rf $TMPDIR && mkdir $TMPDIR"
```

Comment:
I'm doing a lot of extra work here by using `rm`s throughout these files, but I couldnt get Jenkins to scrap my workspaces totally after each build, so there's some dangling state I'd like to get rid of.